scene/2d/camera_2d.cpp

```diff
@@ -554,6 +614,36 @@ Node *Camera2D::get_custom_viewport() const {
 	return custom_viewport;
 }
 
+void Camera2D::set_screen_drawing_enabled(bool enable)
+{
+	screen_drawing_enabled = enable;
+	_update_scroll();
```

Comment:
I think here (and the next two setters as well) you can use just `update()` (from CanvasItem), to force a redraw, no reason to update the position as well.

Not sure how much this applies to various other `_update_scroll()` calls above.